### PR TITLE
Added coinpage with functionality

### PR DESCRIPTION
**Pull request by A00020061**
Added coinpage functionality

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ function App() {
             <Switch>
               <Route path="/" exact component={LandingPage} />
               <Route path="/PortfolioPage" exact component={PortfolioPage} />
-              <Route path="/CoinPage" exact component={CoinPage} />
+              <Route path="/CoinPage/:coinname" exact component={CoinPage} />
               <Route path="/Setting" exact component={Setting} />
```
Comment (yoloOnTheBattlefield) [inline src/App.js:49]: use lowercase like /setting or /coinpage/:coinName

File: src/Components/Charts/charts.styles.js [hunk 2]
```diff
@@ -29,16 +29,17 @@ export const CoinInfoDiv = styled.div`
...
 `;
 export const IndicatorContainer = styled.div`
-  box-shadow: 0px 0px 1px 1px rgba(255, 255, 255, 0.5);
   padding: 0.5rem 1rem;
   position: relative;
+  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.6);
+
   width: 30%;
-  border: solid white;
+  /* border: solid white; */
```
Comment (yoloOnTheBattlefield) [inline src/Components/Charts/charts.styles.js:42]: delete

File: src/services/action/coinNameAction.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { GET_COIN_NAME } from "../reducer/coinNameReducer";
+
+export const getCoinName = (name) => {
+  // console.log("you clicked me just now");
```
Comment (yoloOnTheBattlefield) [inline src/services/action/coinNameAction.js:4]: delete

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/App.js (hunk 1 above)
Comment (A00020061) [reply src/App.js:49]: implemented

**Pull request merged**